daemon/src/setup_maker.rs

```diff
@@ -133,13 +129,28 @@ impl Actor {
     }
 
     async fn complete(&mut self, completed: SetupCompleted, ctx: &mut xtra::Context<Self>) {
-        let _ = self
-            .on_completed
-            .send(completed)
-            .log_failure("Failed to inform about contract setup completion")
-            .await;
+        match self.execute_setup_contract_command(completed).await {
+            Ok(()) => {}
+            Err(e) => {
+                tracing::error!("Failed to execute `contract_setup` command: {:#}", e);
+            }
+        }
+
+        ctx.stop(); // outside of `execute_setup_contract_command` because we always want to shut
+                    // down
```

Comment:
❓ Do we need a comment for this? I thought we had been doing similar stuff elsewhere.